components/json/json_parser_unittest.cc

```diff
@@ -529,7 +529,7 @@ TEST(JsonParser, ConvertAllNumbersToString) {
   // OK: convert floating point values in scientific notation to string
   json = R"({"a": 1.196568750220778e-7})";
   EXPECT_EQ(std::string(json::convert_all_numbers_to_string(json)),
-            R"({"a":"0.0000001196568750220778"})");
+            R"({"a":"1.196568750220778e-7"})");
```

Comment:
Updated serde and serde_json crates. Looks like they preserve scientific notation when converting to a string now. wdyt? @onyb @spylogsster @yrliou

Reply:
reverted all Rust transitive dependency updates

Reply:
the unit test still has a change, is that a mistake?